public/app/features/variables/datasource/DataSourceVariableEditor.tsx

```diff
@@ -108,19 +115,21 @@ export class DataSourceVariableEditorUnConnected extends PureComponent<Props> {
           </VerticalGroup>
 
           <SelectionOptionsEditor
-            variable={this.props.variable}
+            variable={variable}
             onPropChange={this.onSelectionOptionsChange}
-            onMultiChanged={this.props.changeVariableMultiValue}
+            onMultiChanged={changeVariableMultiValue}
           />
         </VerticalGroup>
       </VerticalGroup>
     );
   }
 }
 
-const mapStateToProps: MapStateToProps<ConnectedProps, OwnProps, StoreState> = (state, ownProps) => ({
-  editor: state.templating.editor as VariableEditorState<DataSourceVariableEditorState>,
-});
+const mapStateToProps: MapStateToProps<ConnectedProps, OwnProps, StoreState> = (state, ownProps) => {
+  return {
+    editor: state.templating.editor,
+  };
+};
```

Comment:
While we're in the mood of changing these things we should make sure we follow the new guide lines here:https://github.com/grafana/grafana/blob/main/contribute/style-guides/redux.md#typing-of-connected-props